### PR TITLE
Add example how to create footer with dynamic year

### PR DESCRIPTION
**Pull request by amitaibu**

### CODE REVIEW

File: Guide/recipes.markdown [hunk 1]
```diff
@@ -400,6 +400,53 @@ And then you can style `.hash-target` instead of `:target`:
...
+        currentTime <- getCurrentTime
+        -- Pass it to the default layout.
+        setLayout (defaultLayout currentTime)
+        initAutoRefresh
+```
+
+This means that the `defaultLayout` in the file `Layout.hs` will now get the `UTCtime` as its first argument. Rendering the footer with the copyright could look like this:
+
+
+```hasekll
```
Comment (mpscholten) [inline Guide/recipes.markdown:426]:
```suggestion
```haskell
```

File: Guide/recipes.markdown [hunk 2]
```diff
@@ -400,6 +400,53 @@ And then you can style `.hash-target` instead of `:target`:
...
+instance InitControllerContext WebApplication where
+    initContext = do
+        -- Get the current time.
+        currentTime <- getCurrentTime
+        -- Pass it to the default layout.
+        setLayout (defaultLayout currentTime)
+        initAutoRefresh
+```
+
+This means that the `defaultLayout` in the file `Layout.hs` will now get the `UTCtime` as its first argument. Rendering the footer with the copyright could look like this:
```
Comment (mpscholten) [inline Guide/recipes.markdown:423]:
```suggestion
This means that the `defaultLayout` in the file `Layout.hs` will now get the `UTCTime` as its first argument. Rendering the footer with the copyright could look like this:
```

File: Guide/recipes.markdown (hunk 1 above)
Comment (amitaibu) [inline Guide/recipes.markdown]:
```suggestion
```haskell
```

**Review by mpscholten — APPROVED** [review]

**Comment by mpscholten** [comment]
Thanks :)

**Pull request merged**